client/src/modules/CrossCheck/components/criteria/SubtaskCriteria.tsx

```diff
@@ -1,76 +1,81 @@
-import { Input, Rate, Typography } from 'antd';
+import { Input, Typography, InputNumber, Slider } from 'antd';
+import { useMemo } from 'react';
 import { CrossCheckCriteriaData } from 'services/course';
 
-import { CommentState, CountState } from '../CrossCheckCriteriaForm';
-
 const { TextArea } = Input;
+const { Text } = Typography;
+
 interface SubtaskCriteriaProps {
-  criteriaComment: CommentState[];
   subtaskData: CrossCheckCriteriaData;
-  countStar: CountState[];
-  updateCountStar: (event: number, max: number, key: string) => void;
-  updateComment: (value: string, key: string) => void;
+  updateCriteriaData: (updatedEntry: CrossCheckCriteriaData) => void;
 }
 
-const NUMBER_OF_STARS = 5;
+export function SubtaskCriteria({ subtaskData, updateCriteriaData }: SubtaskCriteriaProps) {
+  const maxScore = subtaskData.max;
+  const comment = subtaskData.textComment;
+  const criteriaScore = subtaskData.point;
+
+  const updateSubtaskData = (newData: number | string) => {
+    const updatedEntry =
+      typeof newData === 'string' ? { ...subtaskData, textComment: newData } : { ...subtaskData, point: newData };
+    updateCriteriaData(updatedEntry);
+  };
```

Comment:
I meant to use something like this :)
```suggestion
  const updateSubtaskData = ({ textComment, point }: { textComment?: string; point?: number }) => {
    const updatedEntry = {
      ...subtaskData,
      ...(isUndefined(textComment) ? undefined : { textComment }),
      ...(isUndefined(point) ? undefined : { point }),
    };
    updateCriteriaData(updatedEntry);
  };
```

Reply:
@andreikovna please fix in a new MR